spec/support/schemas/document_type.json

```diff
@@ -59,7 +59,14 @@
       "type": "array",
       "uniqueItems": true,
       "items": {
-        "$ref": "#/definitions/tag_field"
+        "type": "string",
+        "enum": [
+          "primary_publishing_organisation",
+          "organisations",
+          "role_appointments",
+          "topical_events",
+          "world_locations"
+        ]
```

Comment:
This is cleaner 👍 

One thing I wonder is whether we need to keep this list of allowed ones here - as it's something that needs to be kept in sync. Do we initialise every document type in tests? If so I'd assume the tests would blow up anyway if they had one outside this list

Reply:
Have just tested by adding a made up tag field to publications, and indeed most of the tests blow up with `uninitialized constant DocumentType::FooBarField`. I'll remove this array!